ddl/db_integration_test.go

```diff
@@ -360,8 +399,13 @@ func TestIssue2858And2717(t *testing.T) {
 }
 
 func TestIssue4432(t *testing.T) {
-	store, clean := testkit.CreateMockStore(t)
+	store, dom, clean := testkit.CreateMockStoreAndDomain(t)
 	defer clean()
+
+	ddlChecker := schematracker.NewChecker(dom.DDL())
+	dom.SetDDL(ddlChecker)
+	ddlChecker.CreateTestDB()
```

Comment:
Does this result in normal DDL operations not being tested?

Reply:
ddlChecker will call DDL API for dom.DDL() and SchemaTracker both, so the normal DDL code path is also covered

Reply:
Got it.